examples/table/reactTable/advanced/src/pages/posts/list.tsx

```diff
@@ -171,6 +171,7 @@ export const PostList: React.FC = () => {
         indeterminate,
         ...rest
     }: { indeterminate?: boolean } & React.HTMLProps<HTMLInputElement>) {
+        // eslint-disable-next-line
         const ref = React.useRef<HTMLInputElement>(null!);
```

Comment:
```suggestion
        const ref = React.useRef<HTMLInputElement>(null);
```

Also we should update the below `useEffect` like:

```tsx
        useEffect(() => {
            if (typeof indeterminate === "boolean") {
                if (ref?.current) {
                    ref.current.indeterminate = !rest.checked && indeterminate;
                }
            }
        }, [ref, indeterminate]);
```

Reply:
done